.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDatePeriodParser.cs

```diff
@@ -1738,6 +1739,23 @@ private DateTimeResolutionResult ParseQuarter(string text, DateObject referenceD
             {
                 quarterNum = int.Parse(numberStr);
             }
+            else if (!string.IsNullOrEmpty(orderQuarterStr))
+            {
+                int month = referenceDate.Month;
```

Comment:
where are the logic for relative words? like previous, last, next...

Reply:
Relative info of quarter is in "orderQuarterStr" and then be parsed to swift, then we use the swift to modify the current  quarter num.